src/audio/Playback.ts

```diff
@@ -39,43 +44,25 @@ export interface PlaybackInterface {
     skipTo(timeSeconds: number): Promise<void>;
 }
 
-export const PLAYBACK_WAVEFORM_SAMPLES = 39;
 const THUMBNAIL_WAVEFORM_SAMPLES = 100; // arbitrary: [30,120]
-export const DEFAULT_WAVEFORM = arraySeed(0, PLAYBACK_WAVEFORM_SAMPLES);
 
-function makePlaybackWaveform(input: number[]): number[] {
-    // First, convert negative amplitudes to positive so we don't detect zero as "noisy".
-    const noiseWaveform = input.map((v) => Math.abs(v));
-
-    // Then, we'll resample the waveform using a smoothing approach so we can keep the same rough shape.
-    // We also rescale the waveform to be 0-1 so we end up with a clamped waveform to rely upon.
-    return arrayRescale(arraySmoothingResample(noiseWaveform, PLAYBACK_WAVEFORM_SAMPLES), 0, 1);
-}
-
-export interface PlaybackInterface {
-    readonly currentState: PlaybackState;
-    readonly liveData: SimpleObservable<number[]>;
-    readonly timeSeconds: number;
-    readonly durationSeconds: number;
-    skipTo(timeSeconds: number): Promise<void>;
-}
-
-export class Playback extends EventEmitter implements IDestroyable, PlaybackInterface {
+export class Playback extends EventEmitter implements IDestroyable {
```

Comment:
Do we still need to implement the `PlaybackInterface` here? I can't see the reason for it being removed